### DISCUSSION TITLE
Is Development of this Plugin Dead?

### DISCUSSION THREAD

**Discussion by jonnylink** [prettier/prettier-vscode#3193] [General]
Is this dead? It hasn't worked correctly since July, which is also when the last commit happened. There's also 19 open PRs, many of which have been open for a few months.

Big fan of the plugin, but if it's dead, it's dead. Just need to know if I should give up hope and look for an alternative for our teams.

**Comment by orbiteleven** [comment]
It seems it's taking a nap for sure. Not much action after 31 July '23 when support for Prettier v3 was reverted.

**Comment by ntotten** [comment]
The prettier team has plans to prioritize some work soon. That front said, it works and there isn't much in terms of urgent work.
